sites/public/pages/finder.tsx

```diff
@@ -233,11 +242,13 @@ const Finder = () => {
       setFormData(formCopy)
       if (questionIndex >= formData.length - 1) setIsDisclaimer(true)
       setQuestionIndex(questionIndex + 1)
+      cardBody.current.focus()
```

Comment:
This appears to fix visually what is selected. However, when using the screen reader I don't think it is working correctly. When I click the "next" button the screen reader doesn't jump to the beginning of the card but rather is selected on a non-existent button. If you want I can run through visually what I'm seeing

Reply:
This appears to be working for me, would love to be a part of that convo so I can see what I missed!

Reply:
@ludtkemorgan Could you expand on what you're seeing on your end? The screen recording below is the behavior I'm seeing with a screen reader on 

https://user-images.githubusercontent.com/53269332/201754749-9f029223-63a9-48be-bb80-7de519a0a602.mov

Reply:
I can't get the screen recording to work. There is a chance that I'm just using the screen reader incorrectly. However, If you are available during the huddle I can show you the steps I'm taking to reproduce it. Otherwise I'll try to find a better tool to screen record.